packages/gatsby-plugin-image/src/transforms/gatsby-plugin-image.js

```diff
@@ -30,6 +30,7 @@ const typeMapper = {
 }
 
 export default function jsCodeShift(file, api, options) {
+  // TODO: Allow people to add their babel stuff so parsing always works?
```

Comment:
If users have a `.babelrc.js` or `babel.config.json` file somewhere, it should _already_ work because `parseSync` resolves the config file relative to the file which is being parsed.

Reply:
Oh great. That answers that question :)